Shared/Extensions/DefaultsExtension.swift

```diff
@@ -17,4 +17,6 @@ extension Defaults.Keys {
     static let autoSelectSubtitlesLangCode = Key<String>("AutoSelectSubtitlesLangCode", default: "Auto")
     static let autoSelectAudioLangCode = Key<String>("AutoSelectAudioLangCode", default: "Auto")
     static let appAppearance = Key<String>("appAppearance", default: AppAppearance.system.rawValue)
+    static let videoPlayerJumpForward = Key<Int32>("videoPlayerJumpForward", default: VideoPlayerJumpLength.thirty.rawValue)
+    static let videoPlayerJumpBackward = Key<Int32>("videoPlayerJumpBackward", default: VideoPlayerJumpLength.thirty.rawValue)
```

Comment:
https://github.com/sindresorhus/Defaults#enum-example
`Defaults` supports enum.
You can use this to remove some force unwrapping.

Reply:
https://github.com/jellyfin/Swiftfin/pull/149#discussion_r690927465

Reply:
Thanks for pointing this out. Actually, in order to use this, it would be better to use the Int32 value instead of `length` var as I said I would change [here](https://github.com/jellyfin/Swiftfin/pull/149#discussion_r690931967)

Reply:
I prefer to use `String` rawValue, as in the example above. 
It's your choice.

Reply:
Contextually, this is a length so it makes sense to be a number.

Reply:
> Contextually, this is a length so it makes sense to be a number.
I agree.

Reply:
Also, sorry, I've been swamped with schoolwork already. I'm lucky to get 10 minutes a day to devote to this app now. It may be different when my teachers decide to cool off lol.